docs/backups.md

```diff
@@ -0,0 +1,88 @@
+# Backing up LME Logs
+
+Logs are backed up using the built-in Elastic facilities. Out-of-the-box,
+ElasicSearch supports backing up to filesystems, and this is the only approach
+supported by LME. Other backup destinations are supported but these require
+separate plugins, and are not supported by LME.
+
+## Approach
+
+Backups are created using ElasticSearch snapshots. The initial snapshot will
```

Comment:
Capitalisation, Elasticsearch is a trademark and should only have the E capitalised.